phac-nml/irida-next
# Disable Workflow Executions and Automated Workflow Executions if pipelines.json is empty [DFCT0010085]

## What does this PR do and why?
_Describe in detail what your merge request does and why._

If pipeline.json file is empty:
1. Hide links in the UI (sidebar, projects, groups)
2. Hide launch workflow button (projects and groups)
3. Disable routes

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Update `pipelines.json` file to be an empty array `[]`
2. Restart `bin/dev`
3. Ensure you cannot launch a workflow execution or visit any workflow execution pages.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
